test/e2e/burial-helpers.js

```diff
@@ -0,0 +1,152 @@
+
+  if (data.relationship.type === 'other') {
+    client
+      .fill('input[name="root_relationship_other"]', data.relationship.other)
+      // Not sure what to do with this, exactly, but I'll make it an option.
+      .clickIf('#root_relationship_view:isEntity', data.relationship.isEntity);
+  }
+}
+
+function completeVeteranInformation(client, data) {
+  client
+    .fillName('root_veteranFullName', data.veteranFullName)
+    .fill('input[name="root_veteranSocialSecurityNumber"]', data.veteranSocialSecurityNumber)
+    .fill('input[name="root_vaFileNumber"]', data.vaFileNumber)
+    .fillDate('root_veteranDateOfBirth', data.veteranDateOfBirth)
+    .fill('input[name="root_placeOfBirth"]', data.placeOfBirth);
+}
+
+function completeBurialInformation(client, data) {
+  client
+    .fillDate('root_deathDate', data.deathDate)
+    .fillDate('root_burialDate', data.burialDate)
+    .selectRadio('root_locationOfDeath_location', data.locationOfDeath.location);
+
+  if (data.locationOfDeath.location === 'other') {
+    client.fill('input[name="root_locationOfDeath_other"]', data.locationOfDeath.other);
+  }
+}
+
+function completeServicePeriods(client, data) {
+  data.toursOfDuty.forEach((tour, index) => {
+    client
+      .waitForElementVisible(`input[name="root_toursOfDuty_${index}_dateRange_fromYear"]`, Timeouts.normal)
+      .fillDate(`root_toursOfDuty_${index}_dateRange_from`, tour.dateRange.from)
+      .fillDate(`root_toursOfDuty_${index}_dateRange_to`, tour.dateRange.to)
+      .fill(`input[name="root_toursOfDuty_${index}_serviceBranch"]`, tour.serviceBranch)
+      .fill(`input[name="root_toursOfDuty_${index}_rank"]`, tour.rank)
+      .fill(`input[name="root_toursOfDuty_${index}_serviceNumber"]`, tour.serviceNumber)
+      .fill(`input[name="root_toursOfDuty_${index}_placeOfEntry"]`, tour.placeOfEntry)
+      .fill(`input[name="root_toursOfDuty_${index}_placeOfSeparation"]`, tour.placeOfEntry);
+
+    // Keep adding them until we're finished.
+    if (index < data.toursOfDuty.length - 1) {
+      client.click('.usa-button-outline.va-growable-add-btn');
+    }
+  });
+}
+
+function completePreviousNames(client, data) {
+  if (data.previousNames.length) {
+    client.selectYesNo('root_view:serveUnderOtherNames', true);
+    data.previousNames.forEach((name, index) => {
+      client.fillName(`root_previousNames_${index}`, name);
+
+      if (index < data.previousNames.length - 1) {
+        client.click('.usa-button-outline.va-growable-add-btn');
+      }
+    });
+  } else {
+    client.selectYesNo('root_view:serveUnderOtherNames', false);
+  }
+}
+
+function completeBenefitSelection(client, data) {
+  client
+    .clickIf('input[name="root_view:claimedBenefits_burialAllowance"]', data['view:claimedBenefits'].burialAllowance)
+    .clickIf('input[name="root_view:claimedBenefits_plotAllowance"]', data['view:claimedBenefits'].plotAllowance);
+
+  // TODO: Uncomment this when we get file upload working and testable
```

Comment:
Not sure if this is testable with a mock right now, but I don't know how to do that, so I skipped it for now.